package.json

```diff
@@ -41,7 +41,7 @@
     "codeclimate-test-reporter": "^0.4.0",
     "colors": "^1.1.2",
     "css-loader": "^0.26.1",
-    "d2": "^26.1.7",
+    "d2": "^26.1.12",
```

Comment:
Will give it a try to build-only and see if I manage to run the app. Build error has to do with a test failing. If it ends up being sth tricky, forget about it we can come back to it in the next release.